plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -48,7 +48,13 @@ func (r *RBACAuthorizer) Authorize(requestAttributes authorizer.Attributes) (boo
 	glog.V(2).Infof("RBAC DENY: user %q groups %v cannot %q on \"%v.%v/%v\"", requestAttributes.GetUser().GetName(), requestAttributes.GetUser().GetGroups(),
 		requestAttributes.GetVerb(), requestAttributes.GetResource(), requestAttributes.GetAPIGroup(), requestAttributes.GetSubresource())
 
-	return false, fmt.Sprintf("%v", ruleResolutionError), nil
+	var reason string
+	if ruleResolutionError != nil {
+		reason = fmt.Sprintf("%v", ruleResolutionError)
+	} else {
+		reason = "rules not allow"
```

Comment:
I'm not sure this adds much… maybe just omit the reason if we don't have an error?